ingest/config/config.yaml

```diff
@@ -8,7 +8,7 @@ ncbi_taxon_id:
   general: "11250"
 
 # Params for the transform rulegeneral
```

Comment:
probably want to update the comment too?

Reply:
lol, good catch, will update